### ISSUE TITLE
[latex]: error during loading

### ISSUE THREAD

**Issue by dankessler** [syl20bnr/spacemacs#15467]
After recently updating to emacs 28, I started getting the following warning buffer popping up on opening

```
Error (use-package): tex/:config: Cannot open load file: No such file or directory, tex-buf Disable showing Disable logging
```

I did some digging and tracked it down to `auctex-latexmk`, which (currently) requires a deprecated module from `auctex`. The good news is that this [has already been reported upstream](https://github.com/tom-tan/auctex-latexmk/issues/39) and a [PR is under review](https://github.com/tom-tan/auctex-latexmk/pull/40).

I just wanted to report the issue here as a service to others who might come across the error.

**Comment by dalanicolai** [comment]
As another additional service, a quick solution is to do `SPC f e l auctex-latexmk` and comment out the line `(require 'tex-buf)`. `, e b` to reload the file and `SPC f s` to save it (for later sessions).
`

**Comment by dankessler** [comment]
For a slightly "safer" fix (that should work with older versions of `auctex`, too), consider the very concise patch introduced in [this PR](https://github.com/tom-tan/auctex-latexmk/pull/40/files).

**Comment by dankessler** [comment]
This has been fixed (at least for spacemacs) by #15541
